src/types.ts

```diff
@@ -2,35 +2,36 @@ export type AirtableRecord = {
   source_id: string | null;
   source_name: string | null;
   lead_organization?: string | null;
-  first_author?: string | null;
+  first_author: string | null;
   source_type: string | null;
   test_type: string | null;
   specimen_type: string[] | null;
   isotypes_reported: string[] | null;
-  test_manufacturer?: string[] | null;
-  sensitivity?: number | null;
-  specificity?: number | null;
+  test_manufacturer: string[] | null;
+  sensitivity: number | null;
+  specificity: number | null;
   country: string | null;
-  state?: string[] | null;
-  city?: string[] | null;
+  state: string[] | null;
+  city: string[] | null;
```

Comment:
These aren't optional anymore?

Reply:
no... @atatmaja do you remember the reason why? i forget.

Reply:
Our typing was a bit confusing here, we made the fields optional but also typed them as `string[] | null`. I actually think the best solution is to remove the | null part and just make these optional but haven't tested enough to kniow if that will break anything.

@annoyingpr thoughts??

Reply:
yeah I think making them optional should provide the same function as `string[] | null`!